core/admin/assets/admin-menu-styles.css

```diff
@@ -624,3 +624,41 @@ input.ee-toggle-round-flat:checked + label:after {
     top: -1px;
     margin-right: .25em;
 }
+
+/** ee buttons **/
+.ee-button,
+.ee-button:active,
+.ee-button:visited {
```

Comment:
to avoid any conflicts with other styles, can you plz increase the specificity of these particular rules by prepending them all with `.ee-addon-upsell-info-dv` 

ie:

```css
.ee-addon-upsell-info-dv .ee-button,
.ee-addon-upsell-info-dv .ee-button:active,
.ee-addon-upsell-info-dv .ee-button:visited {
```